runtime/syntax/nftables.yaml

```diff
@@ -0,0 +1,19 @@
+filetype: nftables
+
+detect:
+    filename: "nftables.conf$"
+    header: "^#.*?/etc/nftables"
```

Comment:
I have not used nftables that much but I do not think there would usually be a comment with the filename at the beginning in a ruleset file. I have seen `flush ruleset` being written more but I think changing it like this may be fine:
```suggestion
    header: "^flush +ruleset"
```

Reply:
Yeah it is fine for me. I think it makes more sense for other users